Fix/fix faq ciab descr link

garloff opened the pull request:
In the intro "about", the link to CiaB was broken.
Furthermore, it talked about CiaB as a virtual deployment.
While CiaB can be deployed virtually, it is typically not. The point of CiaB is that it's a single-node deployment.

Being at it:
- Fixing the testbed links.
- Move the mentioning that the ref. impl. for virtualization (IaaS) layer is from OSISM ahead of the deployment examples.
- Mention artcodix and HW labs there as well.
- Mention Cluster-Stacks from syself as ref.impl. as well ahead of the description (to have the same structure)

garloff commented:
Cherry-picked to staging in case you want to view the result ...

berendt commented on docs/index.mdx:
```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
 
-The fastest way to get in touch with SCS is to deploy a SCS cloud virtually. The Cloud-In-A-Box was built explicitly for this scenario. Check it out [here](/docs/iaas/guides/deploy-guide/examples/cloud-in-a-box)
+You can do a single node installation for learning, testing or development purposes.
+The Cloud-In-A-Box configuration was built explicitly for this scenario.
```
Cloud in a Box

garloff replied:
Hmm, while I agree `in a` being lowercase, I consider the term Cloud-in-a-Box a single term rather than four words ... thus my preference to leave the hyphens in it. I believe we have used the form with hyphens at most places in the past. If we want to change that, we should probably grep through our repos and address it everywhere ...

garloff replied:
Partially addressed by 9fc2e2fe.

berendt commented on docs/index.mdx:
```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
+You can do a single node installation for learning, testing or development purposes.
+The Cloud-In-A-Box configuration was built explicitly for this scenario.
+Check it out [here](/docs/iaas/deployment-examples/cloud-in-a-box)
+It comes with a complete set of services, even ceph is part of it
```
Ceph

garloff replied:
Done, see 9fc2e2fe.

berendt commented on docs/index.mdx:
```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
+Read the [testbed docs](/docs/iaas/deployment-examples/testbed) to learn how to get the
 testbed running. Please read carefully through the
-[deployment](https://docs.osism.de/testbed/deployment.html) section of the
+[deployment](https://osism.tech/docs/testbed#deployment) section of the
```
I think it makes more sense to use ` /docs/iaas/deployment-examples/testbed#deployment` here. This way we do not link to an external document.

garloff replied:
Agree, thanks!
Addressed in 9fc2e2fe.

berendt commented on docs/index.mdx:
```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
+
 ### Container Layer
 
+The Reference Implementation (v2) for the container (Kubernetes-as-a-Services = KaaS) layer
```
reference implementation? If Reference Implemention is correct it has to be changed in line 21.

Kubernetes-as-a-Services -> Kubernetes as a Service

garloff replied:
I capitalized Reference Implementation at both places now, giving it the property of a name. See 9fc2e2fe.

garloff replied:
I have not removed the hyphens from Kubernetes-as-a-Service ... I consider this a single word.

berendt commented on docs/index.mdx:
```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
 ### Container Layer
 
+The Reference Implementation (v2) for the container (Kubernetes-as-a-Services = KaaS) layer
+is provided by [Cluster-Stacks](https://docs-staging.scs.community/docs/category/cluster-stacks)
```
Cluster Stacks

berendt replied:
I think this should be `/docs/category/cluster-stacks/`.

garloff replied:
Done, see 9fc2e2fe.

garloff commented:
Thanks for your diligent review, @berendt !
I have addressed most of your suggestions, though not all.
@dirklossack always wanted me to use more hyphens than I did, you seem to be arguing for fewer dashes ...
My personal taste is that when using well-defined terms such as CiaB, KaaS, ... I tend to consider them one word, which is why I run them together into one by using hyphens. I'm certainly not the only one doing this, but I'm happy to learn from folks with better English skills whether or not this is common or a no-go (or both ;-))
